cmd/create/accountroles/cmd.go

```diff
@@ -145,6 +158,12 @@ func run(cmd *cobra.Command, argv []string) {
 		os.Exit(1)
 	}
 
+	// Set HostedCP for Runtime if we are creating HostedCP roles and policies
+	isHostedCP := cmd.Flags().Changed("hosted-cp")
+	if isHostedCP {
+		r.IsHostedCP()
```

Comment:
Not sure if this attribute should be part of the `Runtime` or elsewhere.
Maybe we should have this field added here **accountRolesCreationInput** https://github.com/openshift/rosa/blob/master/cmd/create/accountroles/creators.go#L28 for this command.
@tbrisker tagging you here since you presented the `Runtime` struct.

Reply:
I agree, this doesn't feel like it belongs as part of the `Runtime`